Dockerfile

```diff
@@ -1,11 +1,9 @@
-FROM golang:1.13.4-alpine
+FROM golang:1.15.6-alpine
 
 RUN apk --no-cache add curl git && curl https://glide.sh/get | sh && apk del curl
```

Comment:
@vanastassiou Aren't we trying to remove glide? Do we need the `curl https://glide.sh/get | sh`?

Reply:
You're right, this was a mistake.